CHANGELOG.md

```diff
@@ -45,6 +45,8 @@ terraform values and it will also update the database cluster to use the new con
   - Updated KMS code to aws sdk v3
 - **CUMULUS-2888**
   - Update CloudWatch Events code to AWS SDK v3
+- **CUMULUS-2890**
```

Comment:
Very minor quibble, this should be noted as a breaking change by section, with the removal from the aws-client explicitly called out.    See the CHANGELOG for other examples of what I mean.